requirements/edx/base.in

```diff
@@ -31,7 +31,7 @@ boto==2.39.0                        # Deprecated version of the AWS SDK; we shou
 boto3==1.4.8                        # Amazon Web Services SDK for Python
 botocore==1.8.17                    # via boto3, s3transfer
 celery==3.1.25                      # Asynchronous task execution library
-ddt==0.8.0                          # via xblock-drag-and-drop-v2 (which probably shouldn't require it)
+ddt                                 # Data-Driven Tests: run one test case with different data
```

Comment:
Can this requirement be moved to testing.in?

Reply:
I believe it can. The **ddt package** is used in test units only. It is also used in core libraries directory in the test unit of xblock (which is the one added it to base.in as per the amended comment above)

**_But now we have another issue that causing the [ make upgrade ] to fail, even on new branch from the latest master:_**

**2 days ago** a new version of **pytest-cov (version 2.6.0)** has been released. This package depends on **coverage >= 4.4** which conflicts with the current constraint **coverage==4.2**

Should I add the constraint for **pytest-cov==2.5.1** , or try removing the **coverage** constraint? or should this be a new PR ???

Please advice @nedbat :)

Reply:
- [x] ddt removed from test.in
- [x] pin removed from coverage in coverage.in
- [x] run [ make upgrade ] again

Reply:
I think it's fine too. Coverage should be safe enough for a minor release change.

Reply:
The whole test failed. I missed this in the **pytest-cov** change log: https://pypi.org/project/pytest-cov/2.6.0

Changelog
2.6.0 (2018-09-03)
* Dropped support for **Python < 3.4, Pytest < 3.5** and Coverage < 4.4.

We must stick with the previous version **2.5.1** and pin it in **testing.in** because the project is still on **Python 2.7**

Any other solution @OmarIthawi and @nedbat ?

Reply:
I think that line means "Dropped support for Python 3 < 3.4".  The PyPI page still lists Python 2.7 as supported, and all of the tests passed on Jenkins.